media-sound/audacity/audacity-3.2.2.ebuild

```diff
@@ -0,0 +1,221 @@
+	# From Debian
+	"${FILESDIR}/${PN}-3.2.2-fix-rpaths.patch"
+
+	# Disables some header-based detection
+	"${FILESDIR}/${PN}-3.2.2-allow-overriding-alsa-jack.patch"
+
+	# For has_networking
+	"${FILESDIR}/${PN}-3.2.2-remove-conan-threadpool.patch"
+	"${FILESDIR}/${PN}-3.2.2-allow-finding-rapidjson-pkgconfig.patch"
+)
+
+src_prepare() {
+	cmake_src_prepare
+
+	local header_subs="${S}/lib-src/header-substitutes"
+	cat <<-EOF >"${header_subs}/allegro.h" || die
+	/* Hack the allegro.h header substitute to use system headers.  */
+	#include <portsmf/allegro.h>
+	EOF
+
+	# Keep in sync with has_networking and the ThreadPool.h SRC_URI.
+	if use audiocom; then
+		mkdir -p "${S}/"/lib-src/threadpool/ThreadPool/ || die
+		cp "${DISTDIR}"/progschj-ThreadPool-"${MY_THREADPOOL_DATE}".h \
+		   "${S}"/lib-src/threadpool/ThreadPool/ThreadPool.h || die
+	fi
+}
+
+src_configure() {
+	setup-wxwidgets
+
+	# * always use system libraries if possible
+	# * options listed roughly in the order specified in
+	#   cmake-proxies/CMakeLists.txt
+	# * USE_VST was omitted, it appears to no longer have dependencies
+	#   (this is different from VST3)
+	local mycmakeargs=(
+		-Daudacity_use_nyquist=local
+		#-Daudacity_use_pch leaving it to the default behavior
+		-Daudacity_use_portmixer=$(usex portmixer system off)
+		-Daudacity_use_soxr=system
+
+		-Daudacity_conan_enabled=OFF
+
+		-Daudacity_has_networking=$(usex audiocom on off)
+		# Not useful on Gentoo.
+		-Daudacity_has_updates_check=OFF
+		-Daudacity_has_audiocom_upload=$(usex audiocom on off)
+		-Daudacity_has_threadpool=local
+
+		# The VST3 SDK is unpackaged, and it appears to be under a breed
+		# of a proprietary license and the GPL.
+		-Daudacity_has_vst3=OFF
+		-Daudacity_lib_preference=system
+		-Daudacity_obey_system_dependencies=ON
+		-Daudacity_use_expat=system
+		-Daudacity_use_ffmpeg=$(usex ffmpeg loaded off)
+		-Daudacity_use_libid3tag=$(usex id3tag system off)
+		-Daudacity_use_ladspa=$(usex ladspa)
+		-Daudacity_use_lame=system
+		-Daudacity_use_wxwidgets=system
+		-Daudacity_use_libmp3lame=system
+		-Daudacity_use_libmpg123=$(usex mpg123 system off)
+		-Daudacity_use_libmad=$(usex mad system off)
+		-Daudacity_use_wavpack=$(usex wavpack system off)
+		-Daudacity_use_libogg=$(usex ogg system off)
+		-Daudacity_use_libflac=$(usex flac system off)
+		-Daudacity_use_libopus=$(usex flac system off)
+		-Daudacity_use_libvorbis=$(usex vorbis system off)
+		-Daudacity_use_libsndfile=system
+		-Daudacity_use_portaudio=system
+		-Daudacity_use_midi=system
+		-Daudacity_use_vamp=$(usex vamp system off)
+		-Daudacity_use_lv2=$(usex lv2 system off)
+		-Daudacity_use_portsmf=system
+		-Daudacity_use_sbsms=$(usex sbsms system off)
+		-Daudacity_use_soundtouch=system
+		-Daudacity_use_twolame=$(usex twolame system off)
+
+		# Disable telemetry features.
+		-Daudacity_has_sentry_reporting=off
+		-Daudacity_has_crashreports=off
+
+		# See the allow-overriding-alsa-jack.patch patch
+		-DPA_HAS_ALSA=$(usex alsa on off)
```

Comment:
I'm getting this
```
 * One or more CMake variables were not used by the project:
 *   PA_HAS_ALSA
```
with 
USE:        abi_x86_64 amd64 audiocom elibc_glibc flac kernel_linux mad ogg opus sbsms userland_GNU wavpack

USE:        abi_x86_64 amd64 doc elibc_glibc flac id3tag kernel_linux mpg123 sbsms userland_GNU vamp vorbis wavpack

USE:        abi_x86_64 alsa amd64 elibc_glibc ffmpeg kernel_linux ladspa mad ogg userland_GNU wavpack

First I thought it was related to `-alsa` but clearly it's not.